modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -25,6 +25,7 @@ The following Technology Preview features are enabled by this feature set:
 ** Swap memory on nodes. Enables swap memory use for {product-title} workloads on a per-node basis. (`NodeSwap`)
 ** OpenStack Machine API Provider. This gate has no effect and is planned to be removed from this feature set in a future release. (`MachineAPIProviderOpenStack`)
 ** Insights Operator. Enables the `InsightsDataGather` CRD, which allows users to configure some Insights data gathering options. The feature set also enables the `DataGather` CRD, which allows users to run Insights data gathering on-demand. (`InsightsConfigAPI`)
+** Insights Operator. Enables a new data collection feature called 'Insights Runtime Extractor' which, when enabled, allows Red{nbsp}Hat to gather more runtime workload data about your {product-title} containers. (`InsightsRuntimeExtractor`)
```

Comment:
@michelle-purcell Do we know for certain that the user needs to enable the `TechPreviewNoUpgrade` feature set to use this feature?